agreements/project-contribution/asset-transfer-amendment.md

```diff
@@ -0,0 +1,129 @@
+- [ ] Trademarks ([Schedule E](#schedule-e-trademarks))
+- [ ] Copyrights ([Schedule F](#schedule-f-registered-copyrights))
+- [ ] Other – Describe
+    `_____________________________________________________________________________________________________________`.
+
+## Confirmation
+
+All other terms of the original Agreement remain in effect.
+
+**Assignor Representative:**  
+&nbsp;  
+`By:    ⎣_____________________________________________⎦`  
+`Name:  ⎣_____________________________________________⎦`  
+`Title: ⎣_____________________________________________⎦`  
+`Date:  ⎣_____________________________________________⎦`  
+
+**Sponsor:** *Commonhaus Foundation, Inc.*  
+&nbsp;  
+`By:    ⎣_____________________________________________⎦`  
+`Name:  ⎣_____________________________________________⎦`  
+`Title: ⎣_____________________________________________⎦`  
+`Date:  ⎣_____________________________________________⎦`
+
+## Schedules
+
+### SCHEDULE A: Source code repositories
+
+This schedule identifies repository accounts (e.g., GitHub organizations, individual repositories, and/or bot accounts) that are critical to the continuity of the Project and its operations.
+
+As part of this transfer, the Assignor shall ensure that the Sponsor is granted administrative access to the listed accounts sufficient to support the Project's ongoing operation under the Sponsor's stewardship.
+The Assignor is not required to remove their own administrative access or participation.
+
+- __
+
+&nbsp;
+
+### SCHEDULE B: Domain Names
+
+This schedule identifies domain names necessary for the Project's public identity or technical infrastructure, which are being transferred to the Sponsor.
+
+| Domain name | Registrar |
+| ----------- | --------- |
+| __          | __        |
+
+&nbsp;
+
+### SCHEDULE C: Hosting Accounts
+
+This schedule identifies hosting platforms, infrastructure services, and operational accounts essential to the Project’s functionality and delivery.
+Assignor shall ensure that the Sponsor is granted administrative access sufficient to support ongoing stewardship.
+Assignor is not required to remove their own access or discontinue participation.
+
+- __
+
+&nbsp;
+
+### SCHEDULE D: Social Media and Email Accounts
+
+This schedule identifies social media and email accounts used for public communication or identity of the Project.
+Assignor shall ensure the Sponsor is granted administrative access sufficient to preserve continuity and governance.
+Assignor may retain access or continue co-administration unless otherwise agreed.
+
+- __
+
+&nbsp;
+
+### SCHEDULE E: Trademarks
+
+#### Registered Trademarks
+
+| Trademark | Country | Appln Date | Appln No | Reg Date | Reg No |
+| --------- | ------- | ---------- | -------- | -------- | ------ |
+| __        | __      | __         | __       | __       | __     |
+
+&nbsp;
+
+#### Unregistered Trademarks
+
+(List; should be the same as what is in the [Trademark List][])
```

Comment:
You mean like it needs to be a table like in the Trademark List, or, the contents of this section needs to be the same as in the Trademark List? If the latter, then why not merge both sections?

Reply:
This is a markdown link whose target is defined at the bottom:
https://www.commonhaus.org/trademarks/

The current application process includes adding trademarks to this list (as preliminary / used by permission).